android/src/main/java/com/cyanchill/missingcore/metadataretriever/modules/MetadataRetrieverModule.kt
@@ -174,6 +174,6 @@
-      val isFLAC = uri.endsWith(".flac") || uri.endsWith(".m4a") || uri.endsWith(".mp4")
+      val isFLAC = uri.endsWith(".flac") || uri.endsWith(".m4a") || uri.endsWith(".mp4") || uri.endsWith(".opus") || uri.endsWith(".ogg")

       // Fallback to `MediaMetadataRetriever` if we find nothing with `MetadataRetriever` or with
-      // flac/mp4/m4a files due to artwork not being parsed correctly.
+      // flac/mp4/m4a/opus/ogg files due to artwork not being parsed correctly.
       //  - https://github.com/MissingCore/Music/issues/432

